rest-service/manager_rest/file_storage_manager.py

```diff
@@ -547,8 +548,14 @@ def update_deployment(self, deployment):
         updated_deployment = data[DEPLOYMENTS][deployment_id]
         if deployment.scaling_groups is not None:
             updated_deployment.scaling_groups = deployment.scaling_groups
+        if deployment.updated_at is not None:
+            updated_deployment.updated_at = deployment.updated_at
         self._dump_data(data)
 
+    @staticmethod
+    def _storage_node_id(deployment_id, node_id):
```

Comment:
if you took the time to add this, might as well make other node-related methods use it :)